linode/manager
# refactor: [M3-6308] - MUI v5 Migration - `Components > CreateLinodeDisabled`

## Description 📝

- Removes the `CreateLinodeDisabled` component because it was only used in one place, it does not warrant its own component

## Preview 📷

![Screenshot 2023-04-17 at 4 13 14 PM](https://user-images.githubusercontent.com/115251059/232600047-495f62e9-3806-4b12-a46d-9dfd3cc250c0.jpg)

## How to test 🧪

- Make a restricted user that does not have permission to create linodes
- Visit `http://localhost:3000/linodes/create` and verify you see the notice shown above
